### DISCUSSION TITLE
fa/manager/contribution/How-to-use-API-in-HiddifyManager-project/

### DISCUSSION THREAD

**Discussion by giscus** [hiddify/Hiddify-Manager#4121] [Content Help کمک به اصلاح محتوا]
# fa/manager/contribution/How-to-use-API-in-HiddifyManager-project/

The proxy manager platform.

https://hiddify.com/fa/manager/contribution/How-to-use-API-in-HiddifyManager-project/

**Comment by rbsdotnet** [comment]
درود وقت بخیر نحوه استفاده از API به چه صورت هست
منظور از admin_proxy_path چی هست؟

**Comment by rbsdotnet** [comment]
> # fa/manager/contribution/How-to-use-API-in-HiddifyManager-project/
> 
> The proxy manager platform.
> 
> https://hiddify.com/fa/manager/contribution/How-to-use-API-in-HiddifyManager-project/

برای افزودن ادمین و کاربر از چه API باید استفاده کرد؟
در داخل مستندات پیدا نکردم
در صورت امکان نمونه ای از Url و ساختار درخواست ارسال شود
سپاس

**Reply by rbsdotnet** [reply]
ساخت کاربر ویا ساخت ادمین از طریق API امکان پذیر هست؟

**Reply by ReturnFI** [reply]
بله امکان پذیره

**Comment by rbsdotnet** [comment]
برای ساخت کاربر این خطا رو میده
{
    "msg": "'NoneType' object has no attribute 'name'"
}

**Reply by alintjm** [reply]
سلام داداش... وقتت بخير و خوشى 
براى همين مشكل راه حل پيدا كردى؟
لطفا به من جواب بدى

**Reply by rbsdotnet** [reply]
نه من کلا این پنل و گذاشتم کنار
خیلی ایراد داره و از نظر ساختاری هنوز جای کار داره

**Reply by alintjm** [reply]
my friend isn't available now to translate for me.

after i inspected the source code, i found that there are many problems with many parts of the project, and the api isn't fully implemented/completed, so i agree maybe i have to ignore working with this project.

but are there any alternative project suggestions?!

Thanks for your quick response.

**Reply by TomCat-Hub** [reply]
درود بر شما دوستان عزیز
وقت بخیر

خیلی ممنونم که کامنت گذاشتید و سوالات و مشکلات خودتون رو با ما درمیون گذاشتید.
متاسفانه در حال حاضر مشکلات کوچکی با API وجود داره که در حال رفع و اصلاح اون ها هستید. ممنون میشیم که باگ هایی که باهاشون مواجه میشید رو از طریق لینک زیر ایشو کنید تا توسعه دهندگان پروژه از اون ها باخبر بشن و در اولین فرصت ممکن اون ها رو رفع کنند:
https://github.com/hiddify/Hiddify-Manager/issues

همچنین قویا پیشنهاد میکنم مشکلات خودتون رو توی گروه تلگرام هیدیفای مطرح کنید تا دوستان دولوپر و ساپورت با شما تعاملی سازنده داشته باشند و به شما برای رفع مشکلات کمک کنند.

آدرس لینک گروه تلگرام:
https://t.me/hiddify_board

تک تک گزارشات باگ از طرف تیم هیدیفای برسی میشه و کمک و همراهی شما عزیزان برای ما بسیار ارزشمند و حیاتیست. در اولین فرصت ممکن و در آپدیت های آتی تمامی مشکلات رسیدگی و رفع خواهند شد.

باتشکر از صبر و همراهی شما
مانا باشید!
